options.go

```diff
@@ -49,6 +52,7 @@ type Option func(*Options) error
 
 func GetDefaultOptions() Options {
 	return Options{
+		Logger:      log.NewNopLogger(),
```

Comment:
Would `log.NewDefaultLogger()` be applicable to more use-cases?

Reply:
Generally libraries shouldn't log. It's "bad form."

Reply:
Oh I see what you mean. I thought you were only saying it was bad to not use Go's standard logger interface. Yeah that makes sense.